### ISSUE TITLE
Собрать материалы для курса "Введение на языке Python"

### ISSUE THREAD

**Issue by iradche** [iradche/Data-Expedition-in-EU#17]
https://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-189-a-gentle-introduction-to-programming-using-python-january-iap-2011/download-course-materials/

**Comment by iradche** [comment]
https://ocw.mit.edu/courses/intro-programming/

**Comment by ansakoy** [comment]
ой, а ты хочешь питоновский курс делать? а какого извода питон? и какого плана материалы ты собираешь?
https://learnpythonthehardway.org/book/
http://www.pythonlearn.com/

**Comment by iradche** [comment]
Пока что идея сделать "Gentle introduction to Python".
3.5, видимо.

Спасибо за ссылки!

**Comment by ansakoy** [comment]
по-английски?
если да, то ок. кстати еще куча ссылок на материалы должна быть в комментах к питоновским курсам на stepic.org (там третий). а если по-русски, то на русском языке вообще давно не хватает нормального курса по второму питону, который вполне себе жив. было бы круто хотя бы элементами включить второй в тех местах, где есть расхождения.

**Comment by iradche** [comment]
Да, думаю сделать курс на русском языке. Причем, для непрофильных студентов.
Потом, когда будет фрейм, кину тебе ссылку.
